legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/protocols/pure/vX_X_X/extension/extension_serviceStore.pure

```diff
@@ -45,6 +47,9 @@ function meta::protocols::pure::vX_X_X::extension::store::service::getServiceSto
                    method           = $ser.method,
                    mimeType         = $ser.mimeType,
                    securitySchemes  = $ser.securitySchemes->map(s | $s->meta::protocols::pure::vX_X_X::transformation::fromPureGraph::store::service::transformSecurityScheme($extensions)),
```

Comment:
lets remove securitySchemes from here? If its not possible lets add a TODO that it would be removed later on